### PR TITLE
Develop 1.3.2

### PR DESCRIPTION
**Pull request by michaelkuty**
@glader please can you start from this branch ? or you have unmerged changes ?

this merge contains #56

### CODE REVIEW

**Comment by michaelkuty** [comment]
i separed roadmap, where now is necessary fixes for 1.8 this release is ready for pypi and next will be refactored release with dropped support for dj < 1.4
@jezdez please can you merge this into main develop and release it to pypi as 1.3.2 ?

**Comment by michaelkuty** [comment]
finally tested and works well on django 1.8

**Comment by glader** [comment]
Thank you

File: docs/overview.txt [hunk 1]
```diff
@@ -24,21 +24,41 @@ Setup
...
+        },
+     ]
+
+     Order of ``TEMPLATES.OPTIONS.loaders`` is important. In the former example, templates from database
      will be used as a fallback (ie. when template does not exists in other locations). 
      If you want template from database to be used to override templates in other locations,
-     put ``dbtemplates.loader.Loader`` at beginning of ``TEMPLATE_LOADERS`` settting.
+     put ``dbtemplates.loader.Loader`` at beginning of ``loaders``.
 
 4. Sync your database ``python manage.py syncdb``
```
Comment (g3rd) [inline docs/overview.txt]: Might want to consider:
```
4. Migrate your database ``python manage.py migrate``
```

**Comment by g3rd** [comment]
Is there talk of Django 1.9 support?

**Comment by michaelkuty** [comment]
Initiative has no limits, feel free and make PR into this branch, each forward is welcomed

**Comment by jezdez** [comment]
This was merged into master. develop is no more.

**Comment by marcuspen** [comment]
Has this been released to PyPi?
The latest version seems to only be 1.3 and no sign of 1.3.1, 1.3.2 or 1.4...

**Comment by michaelkuty** [comment]
no it`s only here, but now under jazzband we can publish new release, I think.
i can make this +3days, but if anyone has time, do it please, 1.3.2a1...

**Pull request closed without merging**